Library/ViewModels/ProjectPamphletViewModelTests.swift

```diff
@@ -874,22 +818,11 @@ final class ProjectPamphletViewModelTests: TestCase {
 
       self.scheduler.advance()
 
-      XCTAssertEqual(dataLakeClient.events, ["Page Viewed"])
       XCTAssertEqual(segmentClient.events, ["Page Viewed"])
 
-      XCTAssertEqual(dataLakeClient.properties(forKey: "session_ref_tag"), ["discovery"])
-
-      XCTAssertEqual(dataLakeClient.properties(forKey: "session_user_is_logged_in", as: Bool.self), [true])
```

Comment:
Are we testing this property for segment too?

Reply:
Yes we are. Look inside the `testTrackingProjectPageViewed_LoggedIn()` method